### DISCUSSION TITLE
Remember where month view starts

### DISCUSSION THREAD

**Discussion by jakubasgard** [actualbudget/releases#230] [Feature ideas]
I am not sure if its a bug or feature, so I put it here.
Personally, I do not like that month view always starts in the current month. So now, when I have 3 month view, I see March, April, May.
However, I prefer to also see previous months (to compare actual month against previous ones). No problem, I can move the selection, to start my view on Feb. And see Feb, March, April.
But when I close and reopen the app, I am back to start from March, not Feb. 

Is there any way to make this setup "stick" so its not lost when I close and reopen Actual?

**Comment by Kidglove57** [comment]
Yes - I agree with you! I did raise it prior to GitHub and then got used to it. I now just select the two month view to avoid it annoying me but it would be great if it could be fixed.
